.gitignore

```diff
@@ -1,109 +0,0 @@
-# Byte-compiled / optimized / DLL files
```

Comment:
Any reason this file is deleted?

Reply:
It was not meant to do that, good catch. I have restored it.
Will be there on the updated PR.